### PR TITLE
Increase replicas of internal router to 3.

### PR DESCRIPTION
**Pull request by The-smooth-operator**
I have noticed that internal router has only one replica. We want to have more than 1 replica so when we do rolling restarts of the nodes, the application will still be alive

### CODE REVIEW

File: k8s/templates/router.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ metadata:
   labels:
     app: internal-router
 spec:
-  replicas: 1
+  replicas: 3
```
Comment (fiji-flo) [inline k8s/templates/router.yaml]: Can we template this like for ingress https://github.com/mozilla-iam/dino-park-fence/blob/089628fa32799e155f809dc15661236f3345c024/k8s/values/prod.yaml#L8

Comment (The-smooth-operator) [reply k8s/templates/router.yaml]: makes sense. I've use the occasion to add it to all the deployments

**Comment by The-smooth-operator** [comment]
Note: I've bumped from 1 to 3 replicas in prod gatekeeper and oidc-proxy. Is oidc proxy used?

**Comment by fiji-flo** [comment]
Yes there's one case where it's technically still used. I need to clean this up.

**Pull request merged**